bridges/snowbridge/primitives/router/src/outbound/mod.rs

```diff
@@ -94,13 +96,16 @@ where
 			return Err(SendError::NotApplicable)
 		}
 
+		// Check the location here can only be AssetHub sovereign
+		ensure!(local_sub.len() == 1, SendError::NotApplicable);
 		let para_id = match local_sub.as_slice() {
 			[Parachain(para_id)] => *para_id,
 			_ => {
 				log::error!(target: "xcm::ethereum_blob_exporter", "could not get parachain id from universal source '{local_sub:?}'.");
```

Comment:
```suggestion
				log::debug!(target: "xcm::ethereum_blob_exporter", "only supports Asset Hub root location as the universal source '{local_sub:?}'.");
```

these kind of error logs are spammy for validators, errors should only be used for unexpected situations that need quick attention - this situation is not unexpected/error, just another expected but unsupported case, in such cases we use debug logs useful when debugging